generate-release/src/main.rs

```diff
@@ -93,12 +96,16 @@ fn main() -> anyhow::Result<()> {
             &client,
             overwrite_existing,
         )?,
-        Commands::ReleaseNotes { overwrite_existing } => generate_release_notes(
+        Commands::ReleaseNotes {
```

Comment:
(Nit) Could you add docs here describing that this command creates a bunch of issues? (Also recommend that users add `--dry-run` when testing, so we don't get spammed.)